doc/source/whatsnew/v1.1.0.rst

```diff
@@ -197,6 +197,20 @@ For a full example, see: :ref:`timeseries.adjust-the-start-of-the-bins`.
 
 .. _whatsnew_110.enhancements.other:
 
+fsspec now used for filesystem handling
```

Comment:
Should go above the `.. _whatsnew_110.enhancements.other`. And you might want to add a header for it.

Reply:
I don't think this was fixed. L198 (` .. _whatsnew_110.enhancements.other: `) needs to be below this new section.